src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputPackageDiffDirectory.java

```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.checks;
+
+public class InputPackageDiffDirectory {
```

Comment:
my bad, ignore this request.

BUT two additional:
1) please rename `InputPackageDiffDirectory` to InputPackageDiffDirectoryAtParent
2) please  add a input that is located at com/puppycrawl/tools/checkstyle/checks/coding/InputPackageDiffDirectoryAtSubpackage.java  but package is `package com.puppycrawl.tools.checkstyle.checks.subpackage;`